lib/src/container/skopeo.rs

```diff
@@ -2,9 +2,42 @@
 
 use super::Result;
 use anyhow::Context;
+use serde::Deserialize;
 use std::process::Stdio;
 use tokio::process::Command;
 
+const POLICY_PATH: &str = "/etc/containers/policy.json";
```

Comment:
I think the types here below are mirroring https://github.com/containers/image/blob/main/signature/policy_types.go, let's add a direct link to it.